src/pdf.js

```diff
@@ -28,6 +28,7 @@ if (typeof PDFJS === 'undefined') {
 //#if BUNDLE_BUILD
 //#expand PDFJS.build = '__BUNDLE_BUILD__';
 //#endif
+//#expand PDFJS.documentID = 0;
```

Comment:
I initially created it in api.js. Where specifically can I put it so it is available in fonts.js, and worker.js as well?